src/components/organisms/TimingSpace/TimingSpace.tsx

```diff
@@ -29,9 +29,11 @@ export const TimingSpace: React.FC<TimingSpaceProps> = ({
 }) => {
   const { isShown: selectedSpace, toggle: toggleSelectedSpace } = useShowHide();
 
-  const onClickCreateButton = useCallback(() => setShowCreateEventModal(), [
-    setShowCreateEventModal,
-  ]);
+  const onClickCreateButton = useCallback(() => {
+    const event = { room: spaceName } as WithId<VenueEvent>;
```

Comment:
Per comment here https://github.com/sparkletown/internal-sparkle-issues/issues/1228#issue-1005704209

There is needed to predefine the spaceName only for the "Add an Experience" modal